### PR TITLE
MTL-2237 Fixes and Improvements

### PR DESCRIPTION
**Pull request by rustydb**
These are changes from my recent run of the latest Fawkes code on redbull.

This includes:
- MTL-2237 Fixes
- A new hypervisor Ansible playbook (related to MTL-2247) for configuring NetworkManager on the hypervisors
- New gru and crucible
- Includes Ansible Galaxy install during in `install_ansible`

Depends on:
- https://github.com/Cray-HPE/crucible/pull/24

### CODE REVIEW

File: group_vars/hypervisor/networks.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+---
+networks:
+  interface_defaults:
+    mtu: 9000
```
Comment (manderson-hpe) [inline group_vars/hypervisor/networks.yml:27]: Can these all be 9000?

Comment (manderson-hpe) [reply group_vars/hypervisor/networks.yml:27]: Or maybe, do they need to be 9000?

File: roles/apache2/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+---
+- name: Apache2 templates
+  ansible.builtin.template:
+    mode: '0644'
```
Comment (manderson-hpe) [inline roles/apache2/tasks/main.yml:27]: I thought this wasn't supposed to be in quotes, but even the documentation is inconsistent about it.

**Review by manderson-hpe — APPROVED** [review]

File: group_vars/hypervisor/networks.yml (hunk 1 above)
Comment (rustydb) [reply group_vars/hypervisor/networks.yml:27]: They all should be `9000` for prod, which that's all these are really designed for at the moment.

I could change these to a flat variable name, so dev can more easily override it, e.g.:

```bash
ansible-playbook -e networks_default_mtu=1500
```

File: roles/apache2/tasks/main.yml (hunk 2 above)
Comment (rustydb) [reply roles/apache2/tasks/main.yml:27]: FWIW this was copied from `roles/bootserver/tasks/main.yml` where it already had quotes.
I know what you're getting at though, and I don't remember. I just know what I copied it from was already working.

File: group_vars/hypervisor/networks.yml (hunk 1 above)
Comment (rustydb) [reply group_vars/hypervisor/networks.yml]: I did that, and I added a `vars/fawkes-dev.yml` with a skeleton of a developer's `networks` dictionary.. it's a start.

**Pull request merged**